platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/telemetry/PerformanceEvent.java

```diff
@@ -0,0 +1,68 @@
+package com.mapbox.mapboxsdk.module.telemetry;
+
+import com.mapbox.android.telemetry.Event;
+
+import android.os.Bundle;
+import android.os.Parcel;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+/**
+ * Generic Performance Event that can be used for performance measurements.
+ * Customer measurements can be added to the bundle.
+ */
+public class PerformanceEvent extends Event {
+  private static final String PERFORMANCE_TRACE = "performance.trace";
+
+  private final String event;
+
+  private final String created;
```

Comment:
```suggestion
  private final String eventCreationDate;
```

Maybe rename this variable so it's a little clearer? By its name only, it's not obvious that this variable ends up being a date and is set equal to `DATE_FORMAT.format(new Date())` in the `PerformanceEvent` constructor. Even though `String` is next to the name, it sounds like it could represent some sort of boolean about whether the event is created or not.

Reply:
@langsmith I think this is what we use in other events . Also this matches the name on the backend so we do not have to add a serialization annotation.

Reply:
ahh, ok. Got it.